### PR TITLE
Renames French Fries to Freedom Fries because fuck France

### PR DESCRIPTION
**Pull request by jewishneocon**
duny is french
fuck yu duny

freedom fries

also my gorilla is still stuck i this branch looks like youll have to add it along with patriotism but its okay we all are evolved from gorillas

### CODE REVIEW

**Comment by Boxe** [comment]
i notice this also includes the gorilla

**Comment by jewishneocon** [comment]
it does
its either because i am
shilling hard
dont comprehend github so i fucked up the branch
both

**Comment by clusterfack** [comment]
:us: :+1:

**Comment by Arthurdentist** [comment]
Too easy.
Freedom fries should be their own recipe 

liberty hat + french fries = Freedom Fries

Also nice attempt to sneak in shit via the backdoor.

File: code/modules/mob/living/simple_animal/friendly/gorilla.dm [hunk 1]
```diff
@@ -0,0 +1,20 @@
+//oo oo ah ah
+/mob/living/simple_animal/gorilla
```
Comment (Arthurdentist) [inline code/modules/mob/living/simple_animal/friendly/gorilla.dm]: oh boy its this again woo

**Comment by ComicIronic** [comment]
>renames french fries to freedom fries
>they weren't even called french fries

PR is a lie.

**Comment by Probe1** [comment]
reminder
http://www.youtubemultiplier.com/516cde3d4e79a-sweet-american-baby-jesus.php
reminder

**Comment by BestName** [comment]
Shut up commie

**Comment by PJB3005** [comment]
only making sure people realise fries are belgian and not french

**Comment by d3athrow** [comment]
i thought i already merged gorillas

**Comment by jewishneocon** [comment]
you did pomf this was late at night
i was bored and i decided to make a shitpost pr 
i didnt know how 2git at the time and the gorilla branch was still in it
it was more of a joke than anything this pr

**Comment by SirSkeleto** [comment]
reopen this pr right now

**Comment by d3athrow** [comment]
I can't

**Comment by Boxe** [comment]
excuse me it's been a few minutes since this pr was initially opened

**Pull request closed without merging**